src/components/banner.tsx

```diff
@@ -40,7 +43,7 @@ function Banner() {
   }, []);
 
   const logout = async () => {
-    let url = 'https://movie-test-vercel-delta.vercel.app/delete/logout';
+    let url = `${apiUrl}delete/logout`;
```

Comment:
`apiUrl` ne devrait pas avoir de `/` à la fin, j'ai déjà fait cette manip et quand j'écrivais mes routes ça marchais plus
ça devrait être écrit 
```suggestion
    const url = `${apiUrl}/delete/logout`;
```